src/ur/ur_driver.cpp

```diff
@@ -259,9 +259,11 @@ std::unique_ptr<rtde_interface::DataPackage> urcl::UrDriver::getDataPackage()
   return rtde_client_->getDataPackage(timeout);
 }
 
-bool UrDriver::writeJointCommand(const vector6d_t& values, const comm::ControlMode control_mode)
+bool UrDriver::writeJointCommand(const vector6d_t& values, const comm::ControlMode control_mode,
+                                 const Watchdog& watchdog)
 {
-  return reverse_interface_->write(&values, control_mode);
+  const float read_timeout = verifyWatchdogTimeout(watchdog, control_mode, step_time_);
```

Comment:
Instead of doing this everywhere we could do this inside the reverse_interface inside the `write()` function, right?

Reply:
Yes we could do that, but there are still 3 write functions within the reverse interface, so it wouldn't make a big difference whether we do it in the UrDriver or within the reverse interface.